eZ/Publish/Core/Persistence/Legacy/Content/Gateway/DoctrineDatabase.php

```diff
@@ -1815,6 +1815,45 @@ public function loadRelations($contentId, $contentVersionNo = null, $relationTyp
         return $statement->fetchAll(\PDO::FETCH_ASSOC);
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function countReverseRelations(int $toContentId, ?int $relationType = null): int
+    {
+        $platform = $this->connection->getDatabasePlatform();
+        $query = $this->connection->createQueryBuilder();
+        $expr = $query->expr();
+        $query
+            ->select($platform->getCountExpression('l.id'))
+            ->from('ezcontentobject_link', 'l')
+            ->innerJoin(
+                'l',
+                'ezcontentobject',
+                'c',
+                $expr->andX(
+                    $expr->eq('l.from_contentobject_id', 'c.id'),
+                    $expr->eq('l.from_contentobject_version', 'c.current_version'),
+                    $expr->eq('c.status', 1)
+                )
+            )
+            ->where(
+                $expr->eq('l.to_contentobject_id', ':to_content_id')
+            )
+            ->setParameter(':to_content_id', $toContentId, \PDO::PARAM_INT);
```

Comment:
```suggestion
            ->setParameter('toContentId', $toContentId, ParameterType::INTEGER);
```
(ParameterType might not be imported here yet).